.changeset/hungry-owls-admire.md

```diff
@@ -0,0 +1,34 @@
+---
+"@aws-amplify/ui-react-storage": minor
+"@aws-amplify/ui-react": minor
```

Comment:
Sorry to catch this late, do we need to bump `ui-react`?

Reply:
Nope